util/docker-jenkins/inner/run.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# This file runs inside a WPT testrun container
+# in the context of Travis CI.
```

Comment:
Since this file is for Travis CI-only it shouldn't be named something as general as `run.sh` - how about `travis_ci_run.sh`

Reply:
Done.